README.md

```diff
@@ -1,39 +1,41 @@
 # yay
 
-Yet another Yogurt - An AUR Helper written in Go
+Yet Another Yogurt - An AUR Helper Written in Go
 
 #### Packages
 
 [![yay](https://img.shields.io/aur/version/yay.svg?label=yay)](https://aur.archlinux.org/packages/yay/) [![yay-bin](https://img.shields.io/aur/version/yay-bin.svg?label=yay-bin)](https://aur.archlinux.org/packages/yay-bin/) [![yay-git](https://img.shields.io/aur/version/yay-git.svg?label=yay-git)](https://aur.archlinux.org/packages/yay-git/) [![GitHub license](https://img.shields.io/github/license/jguer/yay.svg)](https://github.com/Jguer/yay/blob/master/LICENSE)  
+
+## Objectives
+
 There's a point in everyone's life when you feel the need to write an AUR helper because there are only about 20 of them.
 So say hi to 20+1.
 
-Yay was created with a few objectives in mind and based on the design of [yaourt](https://github.com/archlinuxfr/yaourt), [apacman](https://github.com/oshazard/apacman) and [pacaur](https://github.com/rmarquis/pacaur):
+_Yay_ is based on the design of [yaourt](https://github.com/archlinuxfr/yaourt), [apacman](https://github.com/oshazard/apacman) and [pacaur](https://github.com/rmarquis/pacaur). It is developed with these objectives in mind:
 
-* Have almost no dependencies
-* Provide an interface for pacman
-* Have yaourt-like search
+* Wrap the _pacman_ interface
+* Search for packages like _yaourt_ does
+* Minimize dependencies
 * Minimize user input
-* Know when git packages are due for an upgrade
+* Know when git packages are due for upgrades
 
 ## Features
 
-* AUR Tab completion
-* Download PKGBUILD from ABS or AUR
-* Ask all questions first and then start building
-* Search narrowing (`yay linux header` will first search linux and then narrow on header)
-* No sourcing of PKGBUILD is done
-* The binary has no dependencies that pacman doesn't already have
-* Advanced dependency solving
+* Perform advanced dependency solving
+* Download PKGBUILDs from ABS or AUR
+* Tab-complete the AUR
+* Query user up-front for all input (prior to starting builds)
+* Narrow search terms (`yay linux header` will first search `linux` and then narrow on `header`)
 * Remove make dependencies at the end of the build process
+* Run without sourcing PKGBUILD
+* Limit runtime dependencies to subset of _pacman_ dependencies
 
 ## Installation
 
-If you are migrating from another AUR helper you can simply install Yay from
-the AUR like any other package.
+If you are migrating from another AUR helper, you can simply install _yay_ with that helper.
 
-The initial installation of Yay can be done by cloning the PKGBUILD and
-building with makepkg.
+If you are starting from scratch, you can install _yay_ by cloning the PKGBUILD and
+then building with _makepkg_:
```

Comment:
I prefer the old wording here.

Reply:
OK, was trying to remove any ambiguity about the two install options being related. I reverted to the original wording for this sentence, but added a leading "Alternatively." Maybe that works.